### ISSUE TITLE
Hackfest schedule: feedback/suggestions

### ISSUE THREAD

**Issue by melissawen** [melissawen/2024linuxdisplayhackfest#11]
Slot distributions proposal (Central European Summer Time - CEST):
![Screenshot from 2024-04-26 10-50-02](https://github.com/melissawen/2024linuxdisplayhackfest/assets/3003316/56f25a2e-379c-4260-bb56-423e80af0bee)

---

### Tue, May 14 (CEST)
- 09:00 - 9:30: Walk to Igalia HQ
- 9:30 - 10:10: Welcome! (Igalia HQ)
- 10:10 - 11:40: Introduction (Participants & Topics)
- 11:40 - 12:00: 20-min break & chat
- 12:00 - 13:30: #6 
- 13:30 - 14:30: Lunch time at HQ
- 14:30 - 16:00: #2
- 16:00 - 16:20: 20-min break & chat
- 16:20 - 18:30: #1
- 20:30: Hackfest Dinner (sponsored by Igalia)

### Wed, May 15 (CEST)
- 09:00 - 9:30: Walk to Igalia HQ
- 9:30 - 10:10: Recap Previous Day (planning)
- 10:10 - 11:40: #5
- 11:40 - 12:00: 20-min break & chat
- 12:00 - 13:30: #4 
- 13:30 - 14:30: Lunch time at HQ
- 14:30 - 16:00: #3
- 16:00 - 16:20: 20-min break & chat
- 16:20 - 18:30: #9

### Thu, May 16 (CEST)
- 09:00 - 9:30: Walk to Igalia HQ
- 9:30 - 10:10: Recap Previous Day (planning)
- 10:10 - 11:40: #7 | #8
- 11:40 - 12:00: 20-min break & chat
- 12:00 - 13:30: #6 | #10
- 13:30 - 14:30: Lunch time at HQ
- 14:30 - 15:10: Wrapping-up
- 15:10 - 18:00:  Social Activity

---

Feel free to report here any clashes or time constraints or better ideas or whatever you think we can organize/do better.
Agenda: https://github.com/melissawen/2024linuxdisplayhackfest/wiki/Topics-&-Talks

**Comment by ppaalanen** [comment]
Please, sprinkle timezone information over the published schedules. The web page had no mention of the timezone at all, and while I can guess the times are local to the venue, I don't remember by heart what timezone that is.

**Comment by melissawen** [comment]
right! I remembered of doing it when creating specific tasks for each session, but I've forgotten to do it here. I just updated the task description: event schedule timezone is CEST. I'll do the same at the event web page.
